### ISSUE TITLE
Organize extension list to categories on https://zed.dev/extensions

### ISSUE THREAD

**Issue by l2D** [zed-industries/extensions#1378]
### Check for existing issues

- [X] Completed

### Misc notes

I'm new to Zed and exploring extensions to extend the  Zed capabilities for my development environment. But it's quite hard to find match extensions only with search functions.

So, I think it would be great if we could organize extensions into collections or categories like themes, integrations, key binding, languages, etc. for exploring interesting extensions and trying them out.

![20240908_0715AM_a4VjjnUk@2x](https://github.com/user-attachments/assets/13733823-fe52-4677-a224-a7b1a5e35832)

**Comment by jere-co** [comment]
Absolutely! And not only on the website, but on the editor itself. Not really productive to browse the extensions when most of them are themes.
